src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.NonNetFx.cs

```diff
@@ -16,13 +16,13 @@ public static partial class PlatformDetection
         public static bool IsNetfx471OrNewer => false;
         public static bool IsNetfx472OrNewer => false;
 
-
+#if !TargetsWindows
```

Comment:
Why are we using ifdefs here rather than putting the relevant declarations into PlatformDetection.Unix.cs?
cc: @danmosemsft